Add attempt at proof. (#122)
HarryFosterCU/Exact-Analysis-of-Heterogeneous-Moran-Processes-in-Climate-Cooperation: tex/main.tex

\]

where $T_{\mathbf{ab}}$ is the transition matrix of the some purely extrinsic dynamic, and
$\delta_{I(\mathbf{a,b})}$ takes the value 1 if $b_{I(\mathbf{a,b})} = C$ and $\frac{1}{2}$
if not. This population dynamic would be a purely extrinsic population dynamic,
but would be tailored to favour a particular action type. We exclude cases such
as this, as they do not conform to the framework in which fitness, is the single
measure of the success of action types in the population, and thus of which
strategy is passed on in each step of the process. While parameters such
as choice intensity and selection intensity affect the transition probability
$T_{\mathbf{ab}}$, they do not inherently favour any specific
action type.

\begin{theorem}[The Moran process is a purely extrinsic process]
\end{theorem}

\begin{proof}
By definition the Moran process satisfies conditions (1) and (3). It remains to
be shown that $T_{\mathbf{ab}}$ is non-decreasing in $\pi_j(\mathbf{a})$ for
$j \in \mathcal{R}$ (strictly increasing for at least one such $j$) and

> non-increasing in $\pi_j(\mathbf{a})$ for $j \notin \mathcal{R}$.

Reviewer: ```suggestion
non-increasing in $\pi_j(\mathbf{a})$ for $j \notin \mathcal{R}$, and that condition (4) holds
```

We need to say that condition (4) holds here

\quad\text{for } j \in \mathcal{R}
$$

For $j \in \mathcal{R}$, as $\epsilon$ is chosen such that $f_i(\mathbf{a}) \gt 0$ for all $i \in
\mathbf{a}$, and $\mu_{i,j}$ such that $\sum_{i=1}^k \mu_{I(\mathbf{a,b}), i} <
1$, this derivative is strictly positive. For $j \notin \mathcal{R}$, the
analogous computation gives
$\frac{dT_{\mathbf{ab}}}{d\pi_j(\mathbf{a})} = -\frac{\epsilon \sum_{x \in \mathcal{R}} f_x(\mathbf{a})}{N(\sum_{l=1}^N f_l(\mathbf{a}))^2}(1 - \sum_{i=1}^k \mu_{I(\mathbf{a,b}), i}) \leq 0$,
so condition (2) holds. Finally, for condition (4), setting
$\pi_i(\mathbf{a}) = v$ for every player $i$ gives $f_i(\mathbf{a}) = 1 + \epsilon v$
for every $i$, whence
$T_{\mathbf{ab}} = \frac{|\mathcal{R}|(1+\epsilon v)}{N\cdot N (1+\epsilon v)}(1 - \sum_i \mu_{I(\mathbf{a,b}), i}) + \frac{\mu_{I(\mathbf{a,b}), b_{I(\mathbf{a,b})}}}{N}
= \frac{|\mathcal{R}|}{N^2}(1 - \sum_i \mu_{I(\mathbf{a,b}), i}) + \frac{\mu_{I(\mathbf{a,b}), b_{I(\mathbf{a,b})}}}{N}$,
which is independent of $v$.
\end{proof}

\begin{theorem}[Fermi imitation dynamics is a purely extrinsic process]
\end{theorem}

\begin{proof}
Once again, by definition Fermi imitation dynamics satisfies conditions (1) and

> (3). It remains to show that it satisfies condition (2).

Reviewer: ```suggestion
(3). It remains to show that it satisfies conditions (2) and (4).
```

Same as in the Moran process

    to mean that the one-step distribution from $\mathbf{a}$ under $P$ is
    stochastically dominated by the one-step distribution from $\mathbf{a}'$
    under $P$ in this order. Let $T^{\mathrm{nd}}_{\mathbf{ab}}$ denote the
    value of $T_{\mathbf{ab}}$ evaluated at $\pi_i(\mathbf{a}) = 0$ for every
    $i$; by condition (4), $T^{\mathrm{nd}}_{\mathbf{ab}}$ is equally the value
    of $T_{\mathbf{ab}}$ under any other choice of common payoff.

    \medskip
    \noindent\textit{Step 1: $T(\mathbf{a}, \cdot) \preceq_{st} T^{\mathrm{nd}}(\mathbf{a}, \cdot)$ for every $\mathbf{a} \in S$.}

    Set $v(\mathbf{a}) := \min_{a_j = D} \pi_j(\mathbf{a})$. In every mixed state
    $\mathbf{a}$, the payoff-dominance hypothesis implies
    $\pi_i(\mathbf{a}) < v(\mathbf{a})$ for every cooperator $i$ and
    $\pi_j(\mathbf{a}) \geq v(\mathbf{a})$ for every defector $j$. Consider a
    $C$-gaining transition $\mathbf{a} \to \mathbf{b}$, so
    $b_{I(\mathbf{a,b})} = C$ and $\mathcal{R}$ is the set of current
    cooperators. Replacing every $\pi_i(\mathbf{a})$ with $v(\mathbf{a})$
    raises all role-model payoffs and lowers all non-role-model payoffs, so by
    condition (2),

    \[

>         T_{\mathbf{ab}} \;\leq\; T_{\mathbf{ab}} \big|_{\pi_i(\mathbf{a}) = v(\mathbf{a})\text{ for all } i},

Reviewer: ```suggestion
        T_{\mathbf{ab}} \;\leq\; T_{\mathbf{ab}} \big|_{\pi_i(\mathbf{a}) = v(\mathbf{a})\text{ for all } i}
```

The , here compiles wierdly

    mutation the mutation matrix is $\sigma$-invariant; and $|\mathcal{R}|$ is
    $\sigma$-invariant because the set of players currently playing
    $b_{I(\mathbf{a,b})}$ in $\mathbf{a}$ has the same size as the set of
    players currently playing $\sigma(b_{I(\mathbf{a,b})})$ in $\sigma(\mathbf{a})$.
    Hence $T^{\mathrm{nd}}_{\mathbf{ab}} = T^{\mathrm{nd}}_{\sigma(\mathbf{a})\sigma(\mathbf{b})}$
    for every transition, so the stationary distribution of $T^{\mathrm{nd}}$
    is $\sigma$-invariant, giving
    $p_C^{(T^{\mathrm{nd}})} = p_D^{(T^{\mathrm{nd}})} = \tfrac{1}{2}$
    \cite{568a8a9c-d8ae-3d24-9fc2-3d33340230e7}.

    \medskip
    \noindent\textit{Step 3: $T^{\mathrm{nd}}$ is stochastically monotone.}

    $T^{\mathrm{nd}}_{\mathbf{ab}}$ depends on $\mathbf{a}$ only through
    $|\mathcal{R}|$, which equals $n_C(\mathbf{a})$ on $C$-gaining transitions
    and $N - n_C(\mathbf{a})$ on $D$-gaining transitions. Combined with the
    label-swap symmetry of Step 2, this implies that $T^{\mathrm{nd}}$
    projects onto $n_C$ as a symmetric birth-death chain on $\{0, 1, \dots, N\}$,
    and so $\mathbf{a} \leq \mathbf{a}'$ implies

>     $T^{\mathrm{nd}}(\mathbf{a}, \cdot) \preceq_{st} T^{\mathrm{nd}}(\mathbf{a}', \cdot)$.

Reviewer: This implication doesn't quite follow to my understanding. We can't say stochastic dominance in this way: take for example this chain:

<img width="586" height="120" alt="Image" src="https://github.com/user-attachments/assets/109b9cd0-3622-4b6d-9e2b-783962d33c73" />

This is technically possible to create under our axioms (within some margin of error for a small self-loop probability). It isn't stochastically monotone, but does still give us our result.
